go.mod

```diff
@@ -3,10 +3,13 @@ module github.com/tendermint/iavl
 go 1.13
 
 require (
+	github.com/golang/protobuf v1.4.0
```

Comment:
We're also importing the `google.golang.org/protobuf` package with the v2 API, it might be better to standardize on that.

Reply:
We're using APIv2 -- `github.com/golang/protobuf` comes from the compiled proto code.

Reply:
I bumped my `protoc-gen-go` to `v1.4.2`.

Reply:
Interesting, wonder why they haven't migrated. Anyway, not a big deal I think, as long as we're using v2 ourselves.

Reply:
They have, I think it's used for different purposes. See https://github.com/golang/protobuf/releases/tag/v1.4.2